charts/incubator/servas/docs/installation_notes.md

```diff
@@ -0,0 +1,29 @@
+# Installation Notes
+
+Generate a cypher key for `App Key` like so.
+
+Click the 3 dot and enter the app shell(select the correct pod).
+
+temp create a .env file by running this command.
+
+```shell
+  echo 'APP_KEY=' >> .env
+```
+
+Generate the key and it will save it in the .env file.
+
+```shell
+php artisan key:generate --force
+```
+
+read the file into shell.
+
+```shell
+cat .env
+```
+
+copy the contents of the output after **APP_KEY=**.
+
+edit the app again and add the contents to App Key in the web gui for servas.
```

Comment:
See firefly how to do that.
aka, generate the key and store it in a secret and set `APP_KEY` as `env`

Reply:
thanks.